src/data_management/rasters/RasterForm.js

```diff
@@ -773,6 +778,64 @@ class RasterFormModel extends Component {
                     />
                   }
                 />
+                <GenericStep
+                  titleComponent={
+                    <FormattedMessage
+                      id="rasters.public_private"
+                      defaultMessage="Who can read"
+                    />
+                  }
+                  subtitleComponent={
+                    <FormattedMessage
+                      id="rasters.public_common_private"
+                      defaultMessage="Specify who can view this raster"
+                    />
+                  }
+                  step={3} // int for denoting which step of the GenericTextInputComponent it refers to
+                  opened={this.props.currentRaster || currentStep === 3}
+                  readonly={false}
+                  currentStep={currentStep} // int for denoting which step is currently active
+                  setCurrentStep={this.setCurrentStep} // cb function for updating which step becomes active
+                  showCheckMark={
+                    this.state.sharedWithOrganisationsWasEverOpenedByUser
```

Comment:
Dit lijkt een copy-paste van question 2.
Is het een idee om hier een aparte variabele voor te maken?
Bijvoorbeeld this.state.whoCanReadWasEverOpenedByUser

Reply:
Heb ik nog niet gedaan. Staat nu direct al een vinkje bij de vraag. Misschien vind ik dit niet zo erg